tests/unit/test_main.py

```diff
@@ -20,3 +22,51 @@ def test_main_with_glob_no_match(mocker):
     execute_side_file_mock = mocker.patch('side_runner_py.main._execute_side_file')
     main.main()
     assert execute_side_file_mock.call_count == 0
+
+
+def test_main_multiple_not_shared(mocker, tmp_path):
+    mocker.patch('side_runner_py.main.with_retry')
+    mocker.patch('side_runner_py.main.get_screenshot')
+    mocker.patch('side_runner_py.main.execute_test')
+
+    # parepare mock test file
+    sidefile_a = tmp_path / "a.json"
+    orig_test_project_a = {
+        'suites': [{'tests': ['foobar_a']}],
+        'tests': [{'id': 'foobar_a', 'commands': []}], 'id': 'foobar_a'
+    }
+    sidefile_a.write_text(json.dumps(orig_test_project_a))
+    sidefile_b = tmp_path / "b.json"
+    orig_test_project_b = {
+        'suites': [{'tests': ['foobar_b']}],
+        'tests': [{'id': 'foobar_b', 'commands': []}], 'id': 'foobar_b'
+    }
+    sidefile_b.write_text(json.dumps(orig_test_project_b))
+
+    # call main with mocked driver, etc...
+    with mock.patch.object(config.sys, 'argv', ['prog_name', '--side-file={}'.format(tmp_path/"*.json")]):
+        main.main()
```

Comment:
Where have the assertions got to?

Reply:
Currently, only checks function does not raise exceptions.

Reply:
uh... well.